fuzz/README.md

```diff
@@ -32,6 +32,34 @@ to the `libFuzzer` library file while configuring; this is represented as
             enable-ssl3 enable-ssl3-method enable-nextprotoneg \
             --debug
 
+Clang uses the gcc libstdc++ library so this must also be installed. You can
+check which version of gcc clang is using like this:
```

Comment:
This still isn't making any sense to me. `apt-cache show clang-14` has this for me:
```
Package: clang-14
[...]
Depends: [...] libstdc++-13-dev, libgcc-13-dev, [...]
```

It should not be possible to install clang without the needed libraries it needs to build things.

Looking here:
https://packages.ubuntu.com/jammy/clang-14

Your version seems to Depend on libstdc++-11-dev, but you claim -12 is needed. I would argue that that is then probably a bug in Ubuntu.

Reply:
Yes, I should probably open a bug with Ubuntu. But on the assumption that it won't get fixed anytime soon, I think this is useful information to have in the README. At least it sucked a non-trivial amount of time while I figured out what was going wrong.